server/server.js

```diff
@@ -111,15 +113,23 @@ function getConfigData(req) {
     userName = req.get('X-Forwarded-User');
     userEmail = req.get('X-Forwarded-Email');
   }
+  let host = process.env.OPENSHIFT_HOST;
+  if (host && host.indexOf('http') === -1) {
```

Comment:
```suggestion
if (host && host.startsWith('http') === -1) {
```

This may be faster, as it does not need to traverse the entire string. This is probably a bit of a micro-optimisation though.